app/src/main/java/org/thoughtcrime/securesms/home/HomeActivity.kt

```diff
@@ -251,7 +251,8 @@ class HomeActivity : PassphraseRequiredActionBarActivity(), ConversationClickLis
             bottomSheet.dismiss()
             val userDetailsBottomSheet = UserDetailsBottomSheet()
             val bundle = Bundle()
-            bundle.putString("publicKey", thread.recipient.address.toString())
+            bundle.putString(UserDetailsBottomSheet.ARGUMENT_PUBLIC_KEY, thread.recipient.address.toString())
+            bundle.putLong(UserDetailsBottomSheet.ARGUMENT_THREAD_ID, thread.threadId)
             userDetailsBottomSheet.arguments = bundle
```

Comment:
There is a nice Bundle builder: `bundleOf(key to value)` that can shave of a few lines when passing arguments

Reply:
That's a good idea, I'll throw that in instead. The present is the perfect time to start improving where we can :smile: